tests/bluetooth/bsim_bt/bsim_test_rpa_peripheral/src/tester.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_bt_utils.h"
+#include "zephyr/bluetooth/addr.h"
+#include "zephyr/bluetooth/conn.h"
+#include <stdint.h>
+#include <zephyr/bluetooth/bluetooth.h>
+
+uint64_t old_time, new_time;
+bt_addr_le_t old_addr;
+bt_addr_le_t *new_addr;
```

Comment:
```suggestion
static volatile bt_addr_le_t old_addr;
static volatile bt_addr_le_t *new_addr;
```

Reply:
Causes compilation warnings (discards 'volatile' qualifier) for bt_addr_le_cmp and print_address.
Should these be ignored ?

Reply:
Nah, just ignore the request for `volatile` here :)